test/integration/test_item_tags.py

```diff
@@ -0,0 +1,143 @@
+from galaxy_test.base.populators import (
+    DatasetCollectionPopulator,
+    DatasetPopulator,
+    WorkflowPopulator,
+)
+from galaxy_test.driver import integration_util
+
+
+class TestHistoriesApi(integration_util.IntegrationTestCase):
```

Comment:
This can be moved to lib/galaxy_test/api/test_tags.py (or its own lib/galaxy_test/api/test_item_tags.py  module).
We usually put API-related tests under `lib/galaxy_test/api/` and under `test/integration` we put tests that involve changing the default Galaxy configuration to something more specific.